src/amo/reducers/guide.js

```diff
@@ -0,0 +1,72 @@
+/* @flow */
+import invariant from 'invariant';
+
+import { createPlatformFiles } from 'core/reducers/addons';
+import type { AddonType } from 'core/types/addons';
+
+export const FETCH_GUIDE_ADDONS: 'FETCH_GUIDE_ADDONS' = 'FETCH_GUIDE_ADDONS';
+export const LOAD_GUIDE_ADDONS: 'LOAD_GUIDE_ADDONS' = 'LOAD_GUIDE_ADDONS';
+
+export type GuideType = {|
```

Comment:
I guess it is the `GuideState`, and this will be useful in the `amo/store` file.

Reply:
It should probably be `GuidesSate` (plural).